src/main/java/it/gov/pagopa/payhub/activities/exception/InvalidFlowDataException.java

```diff
@@ -0,0 +1,8 @@
+package it.gov.pagopa.payhub.activities.exception;
```

Comment:
let's start to partitionate exception package
```suggestion
package it.gov.pagopa.payhub.activities.exception.ingestionflow;
```